.github/workflows/call-webhook.yaml

```diff
@@ -1,7 +1,17 @@
-name: "Jira deployment markers"
+name: "Call webhook"
+
 
 on:
   workflow_call:
+    inputs:
+      ENVIRONMENT:
+        required: false
+        type: string
+        default: integration
+      URL:
+        required: false
+        type: string
+        default: "https://webhook.site/a4328475-c2fe-463a-9478-2c9ceafd7392"
```

Comment:
@SamThing  can we make this required and not set a default value in case one day someone was testing and send sensitive data to it 
what do you think  ?

Reply:
sure, I guess it makes sense